frontend/integration-tests/tests/alertmanager.scenario.ts

```diff
@@ -0,0 +1,457 @@
+
+    // changing url, enables saveAsDefault unchecked, should save pagerduty_url with Receiver
+    await pagerDutyURL.clear().sendKeys('http://pagerduty-url-specific-to-receiver');
+    expect(saveAsDefault.isEnabled()).toBeTruthy();
+    expect(saveAsDefault.getAttribute('checked')).toBeFalsy();
+    await monitoringView.saveButton.click();
+    await crudView.isLoaded();
+
+    //pagerduty_url should be saved with Receiver and not global
+    await horizontalnavView.clickHorizontalTab('YAML');
+    await yamlView.isLoaded();
+    await yamlView.getEditorContent().then((yamlStr) => {
+      const { globals, receiverConfig } = getGlobalsAndReceiverConfig('pagerduty_configs', yamlStr);
+      expect(_.has(globals, 'pagerduty_url')).toBeFalsy();
+      expect(_.get(receiverConfig, 'url')).toBe('http://pagerduty-url-specific-to-receiver');
+    });
+
+    // save pagerduty_url as default/global
+    await browser.get(`${appHost}/monitoring/alertmanagerconfig/receivers/MyReceiver/edit`);
+    await browser.wait(until.presenceOf(firstElementByTestID('cancel')));
+    await pagerDutyURL.clear().sendKeys('http://global-pagerduty-url');
+    await crudView.isLoaded();
+    expect(saveAsDefault.isEnabled()).toBeTruthy();
+    saveAsDefault.click();
+    expect(saveAsDefault.getAttribute('checked')).toBeTruthy();
+    await monitoringView.saveButton.click();
+    await crudView.isLoaded();
+
+    // pagerduty_url should be saved as global, not with Receiver
+    await horizontalnavView.clickHorizontalTab('YAML');
+    await yamlView.isLoaded();
+    await yamlView.getEditorContent().then((yamlStr) => {
+      const { globals, receiverConfig } = getGlobalsAndReceiverConfig('pagerduty_configs', yamlStr);
+      expect(_.get(globals, 'pagerduty_url')).toBe('http://global-pagerduty-url');
+      expect(_.has(receiverConfig, 'url')).toBeFalsy();
+    });
+
+    // save pagerduty url to receiver with an existing global
+    await browser.get(`${appHost}/monitoring/alertmanagerconfig/receivers/MyReceiver/edit`);
+    await browser.wait(until.presenceOf(firstElementByTestID('cancel')));
+    await pagerDutyURL.clear().sendKeys('http://pagerduty-url-specific-to-receiver');
+    expect(saveAsDefault.isEnabled()).toBeTruthy();
+    expect(saveAsDefault.getAttribute('checked')).toBeFalsy();
+    await monitoringView.saveButton.click();
+    await crudView.isLoaded();
+
+    // pagerduty_url should be saved with Receiver, as well as having a global pagerduty_url prop
+    await horizontalnavView.clickHorizontalTab('YAML');
+    await yamlView.isLoaded();
+    await yamlView.getEditorContent().then((yamlStr) => {
+      const { globals, receiverConfig } = getGlobalsAndReceiverConfig('pagerduty_configs', yamlStr);
+      expect(_.get(globals, 'pagerduty_url')).toBe('http://global-pagerduty-url');
+      expect(_.get(receiverConfig, 'url')).toBe('http://pagerduty-url-specific-to-receiver');
+    });
+  });
+
+  it('restores default alertmanager.yaml', async () => {
```

Comment:
Think I have the correct `kubectl patch` cmd, but am getting error msg below.  
Think I have to be `kubeadmin` to patch `alertmanager-mail` Secret

```
>> An error was thrown in an afterAll
   Error: Command failed: kubectl patch secret 'alertmanager-main' --type='json' -n 'openshift-monitoring' -p='[{ op: 'replace', path: '/data/alertmanager.yaml', value: Imdsb2JhbCI6CiAgInJlc29sdmVfdGltZW91dCI6ICI1bSIKInJlY2VpdmVycyI6Ci0gIm5hbWUiOiAibnVsbCIKInJvdXRlIjoKICAiZ3JvdXBfYnkiOgogIC0gImpvYiIKICAiZ3JvdXBfaW50ZXJ2YWwiOiAiNW0iCiAgImdyb3VwX3dhaXQiOiAiMzBzIgogICJyZWNlaXZlciI6ICJudWxsIgogICJyZXBlYXRfaW50ZXJ2YWwiOiAiMTJoIgogICJyb3V0ZXMiOgogIC0gIm1hdGNoIjoKICAgICAgImFsZXJ0bmFtZSI6ICJXYXRjaGRvZyIKICAgICJyZWNlaXZlciI6ICJudWxsIg==}]'
   Error from server (Forbidden): secrets "alertmanager-main" is forbidden: User "dtaylor113" cannot get secrets in the namespace "openshift-monitoring": no RBAC policy matched
```

Reply:
> Think I have the correct `kubectl patch` cmd, but am getting error msg below.
> Do I have to be `kubeadmin` to patch `alertmanager-mail` Secret?
> 
> ```
> >> An error was thrown in an afterAll
>    Error: Command failed: kubectl patch secret 'alertmanager-main' --type='json' -n 'openshift-monitoring' -p='[{ op: 'replace', path: '/data/alertmanager.yaml', value: Imdsb2JhbCI6CiAgInJlc29sdmVfdGltZW91dCI6ICI1bSIKInJlY2VpdmVycyI6Ci0gIm5hbWUiOiAibnVsbCIKInJvdXRlIjoKICAiZ3JvdXBfYnkiOgogIC0gImpvYiIKICAiZ3JvdXBfaW50ZXJ2YWwiOiAiNW0iCiAgImdyb3VwX3dhaXQiOiAiMzBzIgogICJyZWNlaXZlciI6ICJudWxsIgogICJyZXBlYXRfaW50ZXJ2YWwiOiAiMTJoIgogICJyb3V0ZXMiOgogIC0gIm1hdGNoIjoKICAgICAgImFsZXJ0bmFtZSI6ICJXYXRjaGRvZyIKICAgICJyZWNlaXZlciI6ICJudWxsIg==}]'
>    Error from server (Forbidden): secrets "alertmanager-main" is forbidden: User "dtaylor113" cannot get secrets in the namespace "openshift-monitoring": no RBAC policy matched
> ```

Hey @dtaylor113, you run need to be logged in as kubeadmin instead of user dtaylor113. The tests will be kubeadmin in CI.

Reply:
yup, that did the trick.  fixed.